go/consensus/tendermint/full/full.go

```diff
@@ -821,6 +821,17 @@ func (t *fullService) GetStatus(ctx context.Context) (*consensusAPI.Status, erro
 			// We may not be able to fetch the genesis block in case it has been pruned.
 		}
 
+		lastRetainedHeight, err := t.GetLastRetainedVersion(ctx)
```

Comment:
is "version" the same number?

Reply:
Yeah "version" is a generic term used by the MKVS database and it can either represent height (consensus state) or round (runtime state).

Reply:
ok good. couldn't figure out if it was off by one or something. or possibly by more than one if the initial height is not 1